web/Dockerfile

```diff
@@ -35,9 +39,10 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o web/web ./web
 ## package it all up
 FROM gcr.io/linkerd-io/base:2017-10-30.01
 COPY LICENSE /linkerd/LICENSE
-COPY --from=golang /go/src/github.com/linkerd/linkerd2/web .
-RUN mkdir -p ./dist
-COPY --from=webpack-bundle /go/src/github.com/linkerd/linkerd2/web/app/dist ./dist
+COPY --from=golang /go/src/github.com/linkerd/linkerd2/web/web .
```

Comment:
tioli: what do you think about moving `web`, `app`, and `templates` to the `linkerd` top-level directory? that would make it clearer what is our stuff vs. OS stuff, and also avoid any possible top-level naming conflicts going forward.

```dockerfile
## package it all up
FROM gcr.io/linkerd-io/base:2017-10-30.01
WORKDIR /linkerd
COPY LICENSE /linkerd/LICENSE
COPY --from=golang /go/src/github.com/linkerd/linkerd2/web/web /linkerd/web
COPY --from=webpack-bundle /go/src/github.com/linkerd/linkerd2/web/app/dist /linkerd/app/dist
COPY web/templates /linkerd/templates
```

Reply:
This is a great idea! Will definitely do it as part of this branch. It was bugging me that we were dumping everything into the filesystem root.